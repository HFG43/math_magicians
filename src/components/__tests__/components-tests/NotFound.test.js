import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../../notFound';

it('Renders <NotFound /> correctly', () => {
  const notFound = renderer.create(<NotFound />).toJSON();
  expect(notFound).toMatchSnapshot();
});
