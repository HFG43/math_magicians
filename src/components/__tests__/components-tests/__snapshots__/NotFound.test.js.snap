// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <NotFound /> correctly 1`] = `
<div>
  <span>
    Page Not Found
  </span>
</div>
`;
